lib/IRGen/GenMeta.cpp

```diff
@@ -1314,6 +1314,11 @@ namespace {
       if (VTableEntries.empty())
         return;
 
+      // Only emit a method lookup function if the class is resilient
+      // and has a non-empty vtable.
```

Comment:
It only needs this if it's `open`, right?

Reply:
Unfortunately no because methods in extensions can also use super.

Reply:
(I will add tests for that too)

Reply:
Er, right. And I thought explicitly about that on the other one.